packages/service/ni_measurement_plugin_sdk_service/_internal/parameter/_get_type.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any
+
+from google.protobuf.descriptor_pb2 import FieldDescriptorProto
+from google.protobuf.type_pb2 import Field
+
+_TYPE_DEFULAT_MAPPING = {
+    Field.TYPE_FLOAT: float(),
+    Field.TYPE_DOUBLE: float(),
+    Field.TYPE_INT32: int(),
+    Field.TYPE_INT64: int(),
+    Field.TYPE_UINT32: int(),
+    Field.TYPE_UINT64: int(),
+    Field.TYPE_BOOL: bool(),
+    Field.TYPE_STRING: str(),
+    Field.TYPE_ENUM: int(),
+}
+
+_TYPE_FIELD_MAPPING = {
```

Comment:
serialization_descriptors.py imports _TYPE_FIELD_MAPPING, so you should remove the underscore to show that this is part of the public interfaces of the _get_type submodule (which is itself still private)

Or you can add a function wrapping this dict, like you did for get_type_default.